projects/packages/forms/src/contact-form/class-feedback-field.php

```diff
@@ -98,7 +98,7 @@ public function get_label( $context = 'default', $count = 1 ) {
 
 		$postfix = $count > 1 ? " ({$count})" : '';
 
-		if ( 'api' === $context ) {
+		if ( in_array( $context, array( 'api', 'csv' ), true ) ) {
 			if ( empty( $this->label ) ) {
 				return __( 'Field', 'jetpack-forms' ) . $postfix;
 			}
```

Comment:
Just thinking there could be edge cases where forms actually have fields named Field. Probably too much of an edge case to be worth addressing.